ortb/clone_test.go

```diff
@@ -1127,6 +1127,57 @@ func TestCloneDOOH(t *testing.T) {
 	})
 }
 
+func TestCloneBidderReq(t *testing.T) {
+	t.Run("nil", func(t *testing.T) {
+		result := CloneBidderReq(nil)
+		assert.Nil(t, result)
+	})
+
+	t.Run("empty", func(t *testing.T) {
+		given := &openrtb2.BidRequest{}
+		result := CloneBidderReq(given)
+		assert.Equal(t, given, result.BidRequest)
+		assert.NotSame(t, given, result)
+	})
+
+	t.Run("populated", func(t *testing.T) {
+		given := &openrtb2.BidRequest{
+			ID:     "anyID",
+			User:   &openrtb2.User{ID: "testUserId"},
+			Device: &openrtb2.Device{Carrier: "testCarrier"},
+			Source: &openrtb2.Source{TID: "testTID"},
+		}
+		result := CloneBidderReq(given)
+		assert.NotSame(t, given, result.BidRequest, "pointer")
```

Comment:
Do we want an `assert.Equal` here to ensure the values are the same?

Reply:
Pointers should not be the same. We clone BidRequest too: `newReq := ptrutil.Clone(req)` and then User, Device and Source.

Reply:
Oh yes! Values should be the same, but pointers different. Do you want me to add `assert.Equal` for values check?

Reply:
Added